diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -481,6 +481,8 @@ controller_interface::CallbackReturn DiffDriveController::on_deactivate(
   const rclcpp_lifecycle::State &)
 {
   subscriber_is_active_ = false;
+  registered_left_wheel_handles_.clear();
```

Comment:
```suggestion
    if (!is_halted)
    {
      halt();
      is_halted = true;
    }
  registered_left_wheel_handles_.clear();
```

Reply:
oh! I've just pushed, I haven't seen the comment. 
I only called `halt()`, so I'm going to apply your suggestion